### DISCUSSION TITLE
3rd lepton Veto information

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#38] [Comments_HIG_23_001]
Hi @jialin-guo1 ,

We missed adding the 3rd Lepton veto. Please try to check how it affects us.

This should only reduce the contamination from the WZ fully leptonic category. As we didn't check this, I don't know how much this will affect. But for sure, we get some information from the 3rd Lepton veto.

Also, please process and add the WZ fully leptonic sample to the list. Also, add this information here https://github.com/ram1123/2l2Q_limitSettingTool/discussions/25.

**Comment by jialin-guo1** [comment]
Additional loose lepton in resolved SB region in Data
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/fe65f3fd-fe50-4b21-9be4-a42b3b5a46d2)

Additional loose lepton in merged SB region in Data
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/0121a0eb-2ee2-48fd-aa3e-c5a1e0668f69)



in Data SB we see many events with additional leptons. but we don't see larger contamination in SR from this plot https://github.com/ram1123/2l2Q_limitSettingTool/discussions/25#discussioncomment-6430046

**Reply by jialin-guo1** [reply]
I am not sure if is code issue. I will  check my code again

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/c40609e8-bb3e-43fe-8a5a-d97803703abc)

![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/719e6026-2a8e-4ddd-b207-43abe4e18980)

with:
loose lepton pt>10Gev |eta|<2.4

do you think the pt cut is not larger enough?

**Reply by ram1123** [reply]
can you get the numbers? How many % of events have more than 2 loose leptons that passed the selection?

**Reply by jialin-guo1** [reply]
resolved case in Data SB: 4.01% events **have more than 2 loose leptons**
merged case in Data SB: 7.87% events **have more than 2 loose leptons**

resolved case in Data SB: 26.74% events **have more than 1 loose leptons**
merged case in Data SB: 36.55% events **have more than 1 loose leptons**

**Reply by jialin-guo1** [reply]
Do we also need to add this loose lepton veto cut in our analysis?

**Comment by jialin-guo1** [comment]
Significance check by using ggH m1000 signal sample

without veto loose additional loose lepton
S  = n_sig/sqrt(n_sig+n_bkg) = 17.74 **merged**
S  = n_sig/sqrt(n_sig+n_bkg) = 1.07 **resolved**

with veto additional loose lepton(ask event for only two tight leptons, no add additional loose lepton)
S  = n_sig/sqrt(n_sig+n_bkg) = 14.97 **merged**
S  = n_sig/sqrt(n_sig+n_bkg) = 0.88 **resolved**

Significance decrease somehow with with veto additional loose lepton

**Reply by ram1123** [reply]
Also, add the yield of signal and background with and without veto

**Reply by ram1123** [reply]
Also, the WZ fully leptonic sample info.

**Comment by jialin-guo1** [comment]
link WZ information here https://github.com/ram1123/2l2Q_limitSettingTool/discussions/25#discussioncomment-6754788
